src/doc/trpl/match.md

```diff
@@ -20,25 +20,26 @@ match x {
 }
 ```
 
-`match` takes an expression and then branches based on its value. Each *arm* of
-the branch is of the form `val => expression`. When the value matches, that arm's
+`match` takes an expression and then branches based on its value. Each branch or
+*case* is of the form `val => expression`. When the value matches, that case's
```

Comment:
I noticed that in one other place, Rust by Example, quite possibly written by the same person :wink: 

Have to ask: is there any strong reasoning or precedent for this choice? I can somewhat understand if there's a desire to differentiate from `switch` statements for programmers with a C background, but "case" is quite the norm for pattern matching in many other languages and in the literature concerning sum types ("case analysis").